### DISCUSSION TITLE
error in Anura3D file biult by GID

### DISCUSSION THREAD

**Discussion by Aidamehr** [Anura3D/Anura3D_OpenSource#59] [General]
I would like to describe an error I got when I am modeling a slope in GID 16.0 for Anura3D. Although I chose the element type of triangular and the mesh is generated completely, and I considered Anura calculation as 2d plain, I always got this error: Message in line number 20. **** INPUT ERROR: 3D element type is incompatible with chosen dimension (2D - plane strain). ****. I would really appreciate it if you can help me solve the error.
thank you in advance
![image](https://user-images.githubusercontent.com/90531027/211789520-76024139-271e-475f-871c-40afbb5dd747.png)

**Comment by ayerrocolom** [comment]
Hello,
I have never seen this issue before. Probably, this is not an Anura3D issue, but with the definition of the model's geometry.
Can you go to Utilities and click on "Repair Model" and then redo the mesh again?
Also, note that the message says there is an issue with the definition of a line (#23). You should be able to know where this line is if you use the "List entities" tool in GiD.

**Reply by luisez1988** [reply]
Just to add to Prof. Yerro's comment, If you want 2D elements, they will be 2D as long as you apply those to area regions (pink enclosed areas) and your model doesn't have 3D elements (cyan volumetric regions).  

Before creating the mesh go to the mesh menu, then element type, and then select linear triangle elements or 3 noded triangular elements. Select all your model and regenerate the mesh. Usually, the quadratic elements are selected by default, and that will generate an error too.

Hope that helps.

**Reply by luisez1988** [reply]
Also, I think your model has a region without an "area" where you do not have any mesh elements. Create regions for all your model, at the end, everything should have a mesh, even if you don't have material there.
